dkeyserver/Enclave/enclave.cpp

```diff
@@ -494,3 +494,278 @@ int sgx_set_up_tls_server(char *server_port)
         EVP_PKEY_free(pkey);
     return ret;
 }
+
+unsigned long inet_addr2(const char *str)
```

Comment:
why add __2?

Reply:
I try to use the same name with dkeycache because they are same. Or rename it with inet_addr?